src/main/java/uk/gov/hmcts/reform/unspec/callback/CallbackHandler.java

```diff
@@ -9,10 +10,25 @@
 
 public abstract class CallbackHandler {
 
+    private static final String DEFAULT = "default";
+
     protected abstract Map<CallbackType, Callback> callbacks();
 
     public abstract List<CaseEvent> handledEvents();
 
+    public String camundaTaskId() {
+        return DEFAULT;
+    }
+
+    public boolean isEventAlreadyProcessed(BusinessProcess businessProcess) {
+        if (camundaTaskId().equals(DEFAULT)) {
+
+            return false;
+        }
+
+        return businessProcess != null && camundaTaskId().equals(businessProcess.getTaskId());
+    }
+
```

Comment:
again, I thought this was a different story!

Reply:
for time being it is covered in this story